apollo/uma/queries.ts

```diff
@@ -11,6 +11,27 @@ export const ACTIVE_POSITIONS = gql`
           id
         }
       }
+      liquidations {
+        sponsor {
+          id
+        }
+        liquidationId
+        liquidator {
+          address
+        }
+        disputer {
+          address
+        }
+        tokensLiquidated
+        lockedCollateral
+        liquidatedCollateral
+        status
+        events {
+          __typename
```

Comment:
Just curious -- what other types of events are in the liquidation bucket?

Reply:
From the GraphQL schema: 

![Screen Shot 2020-08-05 at 17 09 45](https://user-images.githubusercontent.com/9457025/89464433-7808ac80-d73e-11ea-8eea-86cda5222067.png)